wpeverest/everest-forms
# Column Adjustment Pro feature dependency

### All Submissions:






### Changes proposed in this Pull Request:



This PR allows EVF PRO user to adjust column - show/hide and sort in the entries table.

### How to test the changes in this Pull Request:

1. Create form, Submit few entries.
2. Go To entries table.
3. Click on Setting Cog Icon.
4. Adjust Columns as per needs.
5. Verify the changes.

### Types of changes:




### Other information:




### Changelog entry

> Column Adjustment Pro feature dependency
